solutions/instances/main.tf

```diff
@@ -279,3 +325,23 @@ module "cos_bucket" {
     }
   ]
 }
+
+##############################################################################
+# Event Notification
+##############################################################################
+
+locals {
+  parsed_existing_en_instance_crn = var.existing_en_instance_crn != null ? split(":", var.existing_en_instance_crn) : []
+  existing_en_guid                = length(local.parsed_existing_en_instance_crn) > 0 ? local.parsed_existing_en_instance_crn[7] : null
+  en_region                       = length(local.parsed_existing_en_instance_crn) > 0 ? local.parsed_existing_en_instance_crn[5] : null
+}
+module "event_notification" {
+  count             = var.existing_en_instance_crn != null ? 0 : 1
+  source            = "terraform-ibm-modules/event-notifications/ibm"
+  version           = "1.6.5"
+  resource_group_id = module.resource_group.resource_group_id
+  name              = var.prefix != null ? "${var.prefix}-${var.en_instance_name}" : var.en_instance_name
+  plan              = var.en_plan
+  service_endpoints = var.en_service_endpoint
+  region            = var.en_region
+}
```

Comment:
I think, we should not create EN instance in this DA. Checking more on this...
Also, if we are creating too, we cant use this much minimum configuration of instance.

Reply:
Yes, we should not create EN instance. It should have provision of providing existing en crn if integration is required.